### DISCUSSION TITLE
Ability to disable Create/Add new on Join Field

### DISCUSSION THREAD

**Discussion by ainsleyclark** [payloadcms/payload#8892] [Feature Requests & Ideas]
Originally mentioned in [Discord](https://discord.com/channels/967097582721572934/1298004953951895642/1298005983737679942) it would be great to have the ability in `admin` to disable the use of creating and adding join fields, currently `readOnly` is set to `never` in the type:

```ts
export type JoinField = {
    access?: {
        create?: never;
        read?: FieldAccess;
        update?: never;
    };
    admin?: {
        components?: {
            Error?: CustomComponent<JoinFieldErrorClientComponent | JoinFieldErrorServerComponent>;
            Label?: CustomComponent<JoinFieldLabelClientComponent | JoinFieldLabelServerComponent>;
        } & Admin['components'];
        disableBulkEdit?: never;
        readOnly?: never; // Here
    } & Admin;
}    
```

![image](https://github.com/user-attachments/assets/8010cb49-dd1e-4958-afc0-7ab1dc69903f)

**Comment by DanRibbens** [comment]
This PR adds `admin.allowCreate` which you can set on your join field to prevent adding new documents from the join field: https://github.com/payloadcms/payload/pull/8929

It also solves the awkward bug that you get when you try to create from the join field before you've saved your document and don't have an `id` to relate back to.

This will be released in `3.0.0-beta.121`

**Reply by ainsleyclark** [reply]
Thanks @DanRibbens 🤜
